src/EditorFeatures/Core/Tagging/TaggerUtilities.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.VisualStudio.Text;
+
+namespace Microsoft.CodeAnalysis.Tagging
+{
+    internal static class TaggerUtilities
+    {
+        public static bool SpanEquals(SnapshotSpan? span1, SnapshotSpan? span2, SpanTrackingMode spanTrackingMode)
```

Comment:
>SpanEquals [](http://example.com/codeflow?start=27&length=10)

Seems useful enough to go in SnapshotSpanExtensions.cs

Reply:
i'm fine in either place.

Reply:
So am I, just seemed useful enough and there was already a place with SnapshotSpan extensions.